spec/inputs/jms.yml

```diff
@@ -0,0 +1,18 @@
+---
+# HornetQ Broker
+hornetq:
+  # Connect to a local HornetQ Broker using JNDI
+  :jndi_name: /ConnectionFactory
+  :jndi_context:
+    java.naming.factory.initial: org.jnp.interfaces.NamingContextFactory
+    java.naming.provider.url: jnp://localhost:1099
+    java.naming.factory.url.pkgs: org.jboss.naming:org.jnp.interfaces
+    java.naming.security.principal: guest
+    java.naming.security.credentials: guest
+  :require_jars:
```

Comment:
Indeed the user would be locked to hornetq...I see your point. I think we can move on with the plugin as it is, but in the future we should do something like in the elasticsearch output plugin: have a parent plugin (logstash-input-jms) with all the code and multiple sub plugins (logstash-input-jms-hornetq, logstash-input-jms-activemq, etc.) that set up the jars for that particular JMS provider.